ui/components/ui/button/button.stories.js

```diff
@@ -56,7 +58,19 @@ export default {
 };
 
 export const DefaultStory = (args) => (
-  <Button {...args}>{args.children}</Button>
+  <>
+    <BannerAlert
+      marginBottom={4}
+      severity={SEVERITIES.WARNING}
+      title="Deprecated"
+      description="This version of Button has been deprecated in favour of the component-library version. Contribute to replacing old Button with new Button by submitting a PR against #18896"
```

Comment:
Thanks for all the suggestions! Because of the limits of the component I've decided to remove the reference to the issue and retain the link in the "See details" button. How it looks below. Have also updated the PR description image

<img width="1440" alt="Screenshot 2023-05-02 at 11 12 22 AM" src="https://user-images.githubusercontent.com/8112138/235562188-f907d985-26b7-4b27-af76-6a80216bd883.png">

Updated in https://github.com/MetaMask/metamask-extension/pull/18908/commits/92c5ae0c54c2363f1705b1db629b4f13be3cf31e